pydicom/tests/test_multival.py

```diff
@@ -100,6 +100,14 @@ def testIssue236DeepCopy(self):
         multival = MultiValue(DSfloat, range(7))
         deepcopy(multival)
 
+    def testSorting(self):
```

Comment:
Here you only test the `reverse` parameter. We should also test for the `key` parameter. Or remove the `key` parameter if not needed (at this point)

Reply:
I would want to have it behave like a normal list in respect to sorting, so I want to leave the parameter.
As the parameters are only passed through I thought it unnecessary to test this, but I can add a test, of course.

Reply:
I would add the test

Reply:
Ok, done.

Reply:
@massich - anything left to do here?

Reply:
Sorry I've been absent some days. I think that just adding the test to make sure to catch any change on the behaviour was enough.

Reply:
Thats ok - it is already merged anyway.